mixer/pkg/perf/server.go

```diff
@@ -110,7 +110,7 @@ func initializeArgs(settings *Settings, setup *Setup) (*testEnv.Args, error) {
 }
 
 func (s *server) shutdown() {
-	if s != nil {
+	if s.s != nil {
```

Comment:
just looking at the code snippet...should this be s! nil && s.s != nil ?

Reply:
s should never be nil. s.s could be (hence the original bug).